x-pack/legacy/plugins/siem/public/components/charts/areachart.test.tsx

```diff
@@ -332,20 +323,28 @@ describe('AreaChart', () => {
     },
     customHeight: 324,
   };
+  describe.each(chartDataSets as Array<[ChartSeriesData[]]>)('renders areachart', data => {
+    beforeAll(() => {
+      shallowWrapper = shallow(<AreaChart configs={mockConfig} areaChart={data} />);
+    });
 
-  it('should render if data exist', () => {
-    const mockData = [
-      { key: 'uniqueSourceIps', value: [{ y: 100, x: 100, g: 'group' }], color: '#DB1374' },
-    ];
-    shallowWrapper = shallow(<AreaChart configs={mockConfig} areaChart={mockData} />);
-    expect(shallowWrapper.find('AutoSizer')).toHaveLength(1);
-    expect(shallowWrapper.find('ChartHolder')).toHaveLength(0);
+    it(`with data ${JSON.stringify(data)}`, () => {
+      expect(shallowWrapper.find('AutoSizer')).toHaveLength(1);
+      expect(shallowWrapper.find('ChartHolder')).toHaveLength(0);
+    });
   });
 
-  it('should render a chartHolder if no data given', () => {
-    const mockData = [{ key: 'uniqueSourceIps', value: [], color: '#DB1374' }];
-    shallowWrapper = shallow(<AreaChart configs={mockConfig} areaChart={mockData} />);
-    expect(shallowWrapper.find('AutoSizer')).toHaveLength(0);
-    expect(shallowWrapper.find('ChartHolder')).toHaveLength(1);
-  });
+  describe.each(chartHolderDataSets as Array<[ChartSeriesData[] | null | undefined]>)(
+    'renders prompt',
+    data => {
+      beforeAll(() => {
+        shallowWrapper = shallow(<AreaChart configs={mockConfig} areaChart={data} />);
+      });
+
+      it(`with data ${JSON.stringify(data)}`, () => {
```

Comment:
I'm hesitant to use a `JSON.stringify` just to get a log here. Too expensive for poor readability. maybe just log the index, and if the test fails the engineer can go look at which object it is?

Reply:
like you could log `with data chartHolderDataSets[0]`

Reply:
Good catch, that's definitely a concerned, I tried applying %j from https://jestjs.io/docs/en/api#describeeachtable-name-fn-timeout but didn't work. Will give it another try in the update, otherwise I'll just log it.